src/main/resources/opentext-swagger-ui.css

```diff
@@ -23,7 +22,7 @@
 }
 
 .swagger-ui .topbar a {
-    background-image: url(mf_logo_white.svg);
+    background-image: url(VoltageByOpentextSmallWhite.svg);
```

Comment:
This project is shared across departments.  Updating it to OpenText probably makes sense but not updating it to Voltage.

Reply:
Agreed, this is just a draft and used an image I had handy to test it. Once its working properly I will update to OpenText image instead.